_sass/plain.scss

```
@@ -61,7 +61,9 @@ video {
 
 a {
   color: #000;
-  transition: color 0.3s;
+  text-decoration: underline;
+  text-decoration-thickness: 0.3ex;
```

Actually, 0.2ex is better.
```suggestion
  text-decoration-thickness: 0.2ex;
```